lib/site_prism/section.rb

```diff
@@ -31,23 +30,40 @@ def initialize(parent, root_element, &block)
       within(&block) if block_given?
     end
 
+    ::Capybara::Session::NODE_METHODS.each do |method|
+      class_eval <<~METHOD, __FILE__, __LINE__ + 1
+        def #{method}(...)
+          @root_element.method("#{method}").call(...)
+        end
+      METHOD
+    end
+
+    (::Capybara::Session::DSL_METHODS - ::Capybara::Session::NODE_METHODS).each do |method|
+      class_eval <<~METHOD, __FILE__, __LINE__ + 1
+        def #{method}(...)
+          @parent.method("#{method}").call(...)
```

Comment:
I guess this should be delegated to `Capybara.current_session` instead. @parent is not always a Page class but might be another Section